zappy_server/src/utils/get_item_name.c

```diff
@@ -0,0 +1,30 @@
+/*
+** EPITECH PROJECT, 2023
+** zappy_server
+** File description:
+** get_item_name
+*/
+
+#include "stdlib.h"
+
+char *get_item_name(size_t item)
```

Comment:
Make it a const char *, since we are returning constant strings (in asm, these will be in .text section (or .data?))